docs/opensafely-cli.md

```diff
@@ -0,0 +1,62 @@
+The main tool for using the OpenSAFELY platform locally is the `opensafely` Python module, which is run via the command-line. 
+
+It's main function is to run data extraction and analysis scripts that are specified in the [project pipeline](pipelines.md), in a way that mimics the production environment where real data is accessed. 
+
+It also contains other functions relating to the OpenSAFELY workflow, such as updating codelists from [codelists.opensafely.org](https://codelists.opensafely.org).
+
+## Installing `opensafely`
+
+This is a command-line program.
+
+To install, go to the Anaconda prompt and submit the following command (or use another method to install the module if you know how):
+
+```bash
+pip install opensafely
+```
+
+To check this has installed successfully, submit `opensafely --version`.
+
+## Updating `opensafely`
+
+If you need to install a new version of `opensafely`, update with:
+
+```bash
+pip install --upgrade opensafely
+```
+
+
+## Using `opensafely` at the command line
+
+To view the in-built documentation for each command, submit `opensafely --help` at the terminal, which will list all the ways in which you can use it.
+You can also use `opensafely run --help` to learn more about the `run` command, for example.
+
+To run any of these commands for a specific OpenSAFELY project, you need to change the directory of your prompt to be the repository of the project. 
+For example,  `cd C:/Users/me/my-git-repos/my-repo`.
+
+
+More information on how to use the `opensafely` module is available in specific sections elsewhere, but some key functions are described briefly below.
+
+### `run`
+
+The most common command you'll run. 
+This runs actions defined in the [`project.yaml` file](pipelines.md) and is the main way of testing your code. 
+
+For example,
+
+```bash
+opensafely run make_graph
+```
+
+will run the `make_graph` action.
+
+
+### `codelists`
+This will retrieve each codelist listed in `/codelists/codelists.txt` from [codelists.opensafely.org](https://codelists.opensafely.org), and save them its latest version in the same folder.
+
+Use it like this:
+```bash
+opensafely codelists update
+```
+Running it will add (or update) the codelist `.csv` files in the `codelists/` folder.
+See the [Codelist](codelist-intro.md) section for more information on codelists.
+
```

Comment:
Just a note to say that I've added an `opensafely codelists check` command which will run the same check that we're going to use in the Github Actions tests. This might be useful for people, but then the solution to any issues it finds is always just to run `opensafely codelists update` so I'm not sure if it's worth mentioning at this stage.

Reply:
I've added for completeness